synapseformation/create.py

```diff
@@ -28,6 +28,24 @@ def __init__(self, syn: 'Synapse', only_create: bool = True,
         self.logger = logger or logging.getLogger(__name__)
         self._update_str = "Fetched existing" if only_create else "Created"
 
+    def _find_entity_by_name(self, entity_name: str, parentid: str,
+                             concrete_type: str) -> SynapseCls:
+        """Find an Entity by its name
+
+        Args:
+            entity_name: Name of Entity
+            parentid: Synapse parentid
+            concrete_type: Type of Entity
+
+        Returns:
+            Entity
+        """
+        body = json.dumps({"parentId": parentid, "entityName": entity_name})
+        entity_obj = self.syn.restPOST("/entity/child", body=body)
```

Comment:
This is not a recursive call.  Although I did find the function in the client so I am using that now.  The Jira documentation should definitely be improved, this would be a JIRA ticket against PFLM.